Source/DataSources/ModelGraphics.js

```diff
@@ -179,6 +181,14 @@ define([
          */
         runAnimations : createPropertyDescriptor('runAnimations'),
 
+        /**
+         * Gets or sets the boolean Property specifying if glTF animations should hold the last pose when no frames are specified.
```

Comment:
For consistency make sure that these descriptions are the same. This and above are slightly different.